spec/API_specification/linear_algebra_functions.md

```diff
@@ -145,9 +145,25 @@ TODO
 TODO
 
 (function-matrix_power)=
-### matrix_power()
+### matrix_power(x, n, /)
 
-TODO
+Raises a square matrix (or stack of square matrices) `x` to an integer power `n`.
+
+#### Parameters
+
+-   **x**: _&lt;array&gt;_
+
+    -   input array having shape `(..., M, M)` and whose innermost two dimensions form square matrices. Must have a data type of either `float32` or `float64`.
+
+-   **n**: _int_
+
+    -   integer exponent.
+
+#### Returns
+
+-   **out**: _&lt;array&gt;_
```

Comment:
Allow me to ask a question opposite to what I asked elsewhere: Do we wanna return a namedtuple here?

Reply:
I've updated linalg spec proposals to only return a tuple when more than one array output is returned. In this case, as only one array is returned, seems okay to leave as is.